staging/operator-lifecycle-manager/pkg/controller/operators/catalog/operator.go

```diff
@@ -600,6 +602,23 @@ func NewOperator(ctx context.Context, kubeconfigPath string, clock utilclock.Clo
 		}
 	}
 
+	// Wire NetworkPolicies
```

Comment:
nit: This could say something more useful.

Reply:
I'll try to resolve this one upstream and we can bring it down through auto-sync. Don't want to hold this one back.